packages/manager/modules/billing/src/payment/method/add/index.html

```diff
@@ -87,10 +87,45 @@
                                         data-translate="billing_payment_method_add_selected_paypal_redirect"
                                     ></span>
                                 </p>
+                                <div
+                                    data-ng-switch-when="SEPA_DIRECT_DEBIT"
```

Comment:
**thought**
It could be interesting to split each switch case in templates, to avoid having to much html in one file.